test/functional/rpc_blockchain.py

```diff
@@ -307,6 +309,15 @@ def assert_waitforheight(height, timeout=2):
         assert_waitforheight(current_height)
         assert_waitforheight(current_height + 1)
 
+    def _test_getblockbyheight(self):
+        self.log.info("Test getblockbyheight")
+        node = self.nodes[0]
+        node.add_p2p_connection(P2PInterface())
+
+        assert_equal(node.getblock(node.getblockhash(20)), node.getblockbyheight(20))
+        self.log.info("The block is valid")
```

Comment:
No need for the `The block is valid` logging, or the extra white space here.

Reply:
Fixed